samples/tab-stage-view/csharp/TabInStageView/README.md

```diff
@@ -0,0 +1,69 @@
+﻿# TabInStageView
+
+Bot Framework v4 echo bot sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple bot that accepts input from the user and echoes it back.
```

Comment:
Wrong Summary
**"accepts input from the user and echoes it back."**

Kindly add proper summary.